air_sim.h

```diff
@@ -9,7 +9,7 @@
 #define MAX 15
 #define ARRIVE 0
 #define DEPART 1
-#define RAND_MAX 
+#define RAND_MAX  1

```

Comment:
Not required

Reply:
Ah sorry actually it was showing an error in my editor , thats why I had added it.

Reply:
I'll change it now

Reply:
Yeah, pls complete initairport func also so u can get more points!!!